.gitignore

```diff
@@ -102,3 +102,7 @@ dist
 
 # TernJS port file
 .tern-port
```

Comment:
You can remove other ingores from this list as those are mostly specific to NPM projects and were just generated automatically.

Reply:
You suggest to remove the automated ones or the custom one I had at the end? I kept the automated ones as they were covering the specific one and they do not create any conflict.